### ISSUE TITLE
Pack all std note styles, distribute once, switch by script

### ISSUE THREAD

**Issue by purple4pur** [purple4pur/osu-skins#1]
Maybe it is possible to pack all std note styles in a folder like this:

```
skin top-level folder
 |- flavor
 |   |- style1
 |   |   |- cursor.png
 |   |   |- ...
 |   |- style2
 |   |   |- ...
 |   |- something runnable (script)
 |- skin.ini
 |- ...
```

By running the script with parameters, it can simply remove un-needed files from top-level and place one style into the top-level. So a user can switch among flavors in a single `.osk` file, without changing files manually.

The script can be written in pure bat? or (lua executable + lua script + a entry bat)?

**Comment by purple4pur** [comment]
Implemented in ce4d127 . Released with [v3.2.0](https://github.com/purple4pur/osu-skins/releases/tag/v3.2.0) .

**Comment by purple4pur** [comment]
Implement with pure bat for the best compatibility.
